src/pages/library/DataBrowserDetails.js

```diff
@@ -223,10 +261,81 @@ export const SidebarComponent = ({ dataObj, id }) => {
     showRequestAccessModal && h(RequestDatasetAccessModal, {
       datasets: [dataObj],
       onDismiss: () => setShowRequestAccessModal(false)
+    }),
+    datasetNotSupportedForExport === true && h(Modal, {
+      title: 'Cannot Export Dataset',
+      showCancel: false,
+      onDismiss: () => setDatasetNotSupportedForExport(false)
+    }, ['This dataset is not hosted in a storage system that currently has the ability to export to a research environment.']),
+    !!snapshotExportJobId && h(SnapshotExportModal, {
+      jobId: snapshotExportJobId,
+      dataset: dataObj,
+      onDismiss: () => {
+        setSnapshotExportJobId()
```

Comment:
Nit: IntelliJ is saying that an argument is expected. We should probably pass `undefined` here and on line 279 (since that's the default from `useState`).

Reply:
Interesting. I did not get that message from intellij. I wonder if it is a configurable setting. Either way, maybe the current team on the terra-ui management rotation can take a ticket to add that to the linter?